doc/7/controllers/security/get-user-strategies/index.md

```diff
@@ -0,0 +1,37 @@
+---
+code: true
+type: page
+title: getUserStrategies
+description: Gets all the available authentication strategies of a user
+---
+
+# getUserStrategies
```

Comment:
missing "since" badge

(perhaps a "since kuzzle version xxx" should be added too?)